cmd/web-handlers.go

```diff
@@ -198,6 +198,21 @@ func (web *webAPIHandlers) MakeBucket(r *http.Request, args *MakeBucketArgs, rep
 		return toJSONError(ctx, errBucketAlreadyExists)
 	}
 
+	if globalOperatorDNSConfig != nil {
+		// Proceed to creating a bucket.
+		if err := objectAPI.MakeBucketWithLocation(ctx, args.BucketName, opts); err != nil {
```

Comment:
The operator is idempotent if the same bucket is created twice it returns 200, it might be better to create the service and then create the bucket. This way, there is no clean up needed.